components/style/properties/properties.mako.rs

```diff
@@ -2048,6 +2048,22 @@ pub fn modify_style_for_anonymous_table_object(
     let box_style = Arc::make_mut(&mut style.box_);
     box_style.display = new_display_value;
     box_style.position = longhands::position::computed_value::T::static_;
+
+    let padding = Arc::make_mut(&mut style.padding);
```

Comment:
@Ms2ger Mmh there is no direct reference I can find saying that anonymous table elements should not inherited style from their parents. On the other hand there is also no reference saying that they _should_ inherited visual style, afaik.